setup.py

```diff
@@ -1,22 +1,31 @@
-from setuptools import setup
+from setuptools import find_namespace_packages, setup
 
-requires = [
+install_requires = [
     "click>=7.1.2",
     "requests>=2.24.0",
+    "jsonschema>=4.17.0",
+]
+
+tests_require = [
+    "pytest",
+]
+
+dev_require = [
+    *tests_require,
+    "black",
+    "click-man",
+    "flake8",
+    "isort",
+    "mypy",
+    "tox",
+    "types-click",
+    "types-requests",
+    "types-jsonschema",
```

Comment:
I'm not familiar with how setup.py extras_require works, but all of these dependencies (except for Tox itself) appear in a deps= line of some Tox environment. Should tests_require list these instead, and dev_require only include *tests_require and Tox itself?

Reply:
`test_requires` should only specify what is needed to run the tests in the `tests/` directory. For that, you only need pytest, Tox is just a wrapper on top of that to make it easy for devs. Everything else is then a dev dependency (linting doesn't count as testing in the traditional packaging world.)

Reply:
"linting doesn't count as testing in the traditional packaging world" - odd, but ack. I was thinking of everything that might run in a Gitlab CI pipeline / Tox job as a "test" that I'd want to pass before merging code, building a package or doing a release. Thanks for explaining.